### ISSUE TITLE
Bazel and Preview.js

### ISSUE THREAD

**Issue by KatSick** [fwouts/previewjs#1503]
### Describe the bug

We are using Bazel which uses symlinks aggressively in `dist` folder:
```
500 in endpoint /previews/start:
Error: ELOOP: too many symbolic links encountered, stat '/Users/ostapchervak/Projects/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/redacted/dist/bin/somemodule/generate-tsconfig-test.update.runfiles/someproject/somemodule/generate-tsconfig-test.update'
```

### Reproduction

1. Create some rules_nodejs project
2. Build app
3. Run preview.js

### Preview.js version

v1.19.6

### Framework

React 18

### System Info

```shell
macOS 13.2.1 (arm64 Darwin 22.3.0). M1 Pro. Node@16, Yarn@3, Bazel@6, rules_nodejs@5
```

### Used Package Manager

yarn

**Comment by fwouts** [comment]
Thank you @KatSick! I can confirm this is easily reproducible by creating the following directory structure:

```
dist/
  foo/
    dist = symlink to dist/ above
```

Will aim to fix for the next release.

**Comment by fwouts** [comment]
This was an easy fix, thanks again for making me aware of the issue! 🥳

**Comment by KatSick** [comment]
wow! thank you for the fix! is there any way to try how it works now?

**Comment by fwouts** [comment]
@KatSick I'm planning to make a release tomorrow!

You can also try building it yourself (see https://github.com/fwouts/previewjs/blob/main/CONTRIBUTING.md#working-with-the-vs-code-extension) but note this will have a much simpler UX than the official release.

**Comment by fwouts** [comment]
Hey @KatSick, the fix is now live in v1.20!

Let me know how you go, feel free to reopen the issue if it's not addressed.
